index.html

```diff
@@ -38,5 +38,95 @@ <h1> <span class="wrapper">I'm Mariane</span> <span class="wrapper">Glad to see<
             </ul>
          </div>
     </section>
+    <main>
+        <section class="card-1">
+            <div class="card-1-bg"></div>
+            <h2>Tonic</h2>
+            <div class="canopy"></div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="lang-btn">
+               <ul>
+                  <li><a href="#"><button>html</button></a></li>
+                  <li><a href="#"><button>css</button></a></li>
+                  <li><a href="#"><button>javaScript</button></a></li>
+               </ul>
+            </div>
+            <div class="card-1-btn"><button>See Project</button></div>
+        </section>
```

Comment:
- [ ] For good semantics reasons I think it would be better of you use a `div` tag as container for each project card instead of using the `section` tag. The `section` tag should wrap all the cards